ScreenBoom/ScreenBoom/View/RightMenuView.swift

```diff
@@ -86,38 +97,91 @@ class RightMenuView: UIView {
                       trailing: trailingAnchor,
                       padding: .init(top: 84, left: 10, bottom: 0, right: 10),
                       size: .init(width: 0, height: 30))
-    playImage.anchor(top: editButton.bottomAnchor,
+    playButton.anchor(top: editButton.bottomAnchor,
                      leading: leadingAnchor,
                      bottom: nil,
                      trailing: trailingAnchor,
                      padding: .init(top: 15, left: 20, bottom: 0, right: 20),
                      size: .init(width: 0, height: 40))
-    pauseImage.anchor(top: playImage.bottomAnchor,
+    pauseButton.anchor(top: playButton.bottomAnchor,
                       leading: leadingAnchor,
                       bottom: nil,
                       trailing: trailingAnchor,
                       padding: .init(top: 10, left: 20, bottom: 0, right: 20),
                       size: .init(width: 0, height: 40))
-    deleteImage.anchor(top: pauseImage.bottomAnchor,
+    deleteButton.anchor(top: pauseButton.bottomAnchor,
                        leading: leadingAnchor,
                        bottom: nil,
                        trailing: trailingAnchor,
                        padding: .init(top: 10, left: 20, bottom: 0, right: 20),
```

Comment:
okay so we dont like to have random numbers keda.. make a constants object or even beter a struct named SideMenuLayout (same thing as a constants struct just a different name) and add all of these values to it. You can put it at the top of this class